repose-aggregator/commons/commons-configuration/src/test/resources/unmarshallerValidator/xsd/uri-stripper.xsd

```diff
@@ -19,23 +19,29 @@
   =_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
   -->
 
-
-<xs:schema xmlns:uri-stripper="http://docs.openrepose.org/repose/uri-stripper/v1.0" xmlns:html="http://www.w3.org/1999/xhtml"
+<xs:schema targetNamespace="http://docs.openrepose.org/repose/uri-stripper/v1.0"
```

Comment:
I removed this in #1555. That will likely get merged first, and this will be cleared up.